src/css/profile/mobile/common/appbar.less

```diff
@@ -219,4 +220,20 @@
 			overflow: hidden;
 		}
 	}
+
+	.ui-checkbox-all {
+		font-size: 12 * @sp_base;
+		width: 32 * @px_base;
+		display: flex;
+		flex-direction: column;
+		align-items: center;
+		padding-top: 10 * @px_base;
+		line-height: 10 * @px_base;
+		margin-left: 18 * @px_base;
+		margin-right: 18 * @px_base;
```

Comment:
If there is no margin for the top and bottom, it would be good to use the 'margin: 0 18*px_base'?

Reply:
I am against setting redundant styles. TAU selectors are very strong and many times are hard to overwritten. If I don't have to, I try not to create them.

Reply:
I agree it!